src/WebJobs.Script.WebHost/WebHooks/WebJobsSdkExtensionHookProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Azure.WebJobs.Script.WebHost.Controllers;
+using HttpHandler = Microsoft.Azure.WebJobs.IAsyncConverter<System.Net.Http.HttpRequestMessage, System.Net.Http.HttpResponseMessage>;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    // Gives binding extensions access to a http handler. 
+    // This is registered with the JobHostConfiguration and extensions will call on it to register for a handler. 
+    internal class WebJobsSdkExtensionHookProvider : IWebhookProvider
```

Comment:
Normalize casing in IWebhook to IWebHook. We use the latter throughout, and also the ASP.NET APIs use that